pkg/registry/rbac/rest/storage_rbac.go

```diff
@@ -139,8 +139,8 @@ func (p RESTStorageProvider) storage(version schema.GroupVersion, apiResourceCon
 
 func (p RESTStorageProvider) PostStartHook() (string, genericapiserver.PostStartHookFunc, error) {
 	policy := &PolicyData{
-		ClusterRoles:               append(bootstrappolicy.ClusterRoles(), bootstrappolicy.ControllerRoles()...),
-		ClusterRoleBindings:        append(bootstrappolicy.ClusterRoleBindings(), bootstrappolicy.ControllerRoleBindings()...),
+		ClusterRoles:               append(append([]rbacapiv1.ClusterRole{}, bootstrappolicy.ClusterRoles()...), bootstrappolicy.ControllerRoles()...),
+		ClusterRoleBindings:        append(append([]rbacapiv1.ClusterRoleBinding{}, bootstrappolicy.ClusterRoleBindings()...), bootstrappolicy.ControllerRoleBindings()...),
```

Comment:
these two methods return a fresh copy of roles/rolebindings on each invocation, no copy should be required